test/e2e/framework/service/jig.go

```diff
@@ -812,8 +812,15 @@ func testEndpointReachability(endpoint string, port int32, protocol v1.Protocol,
 		e2elog.Failf("Service reachablity check is not supported for %v", protocol)
 	}
 	if cmd != "" {
-		_, err := framework.RunHostCmd(execPod.Namespace, execPod.Name, cmd)
-		framework.ExpectNoError(err, "Service is not reachable on following endpoint %s over %s protocol", ep, protocol)
+		err := wait.PollImmediate(framework.PollShortTimeout, ServiceReachabilityShortPollTimeout, func() (bool, error) {
```

Comment:
Can we use a shorter interval here? `PollShortTimeout` is 1 minute, which meas we will try two times at most.

Reply:
Thanks for pointing. I was intended to do for 1 sec.